autogen/agentchat/groupchat.py

```diff
@@ -232,17 +236,35 @@ def select_speaker_msg(self, agents: Optional[List[Agent]] = None) -> str:
         """Return the system message for selecting the next speaker. This is always the *first* message in the context."""
         if agents is None:
             agents = self.agents
-        return f"""You are in a role play game. The following roles are available:
-{self._participant_roles(agents)}.
 
-Read the following conversation.
-Then select the next role from {[agent.name for agent in agents]} to play. Only return the role."""
+        roles = self._participant_roles(agents)
+        agentlist = f"{[agent.name for agent in agents]}"
+
+        if self.custom_select_speaker_msg is None:
+            # Standard select speaker message
+            return f"""You are in a role play game. The following roles are available:
```

Comment:
We can make this string a default value for the argument.

Reply:
Yep, sounds good, will do